reference/shaders-no-opt/comp/subgroups_basicvoteballot.vk.comp

```diff
@@ -284,6 +284,51 @@ void subgroupMemoryBarrierBuffer() { groupMemoryBarrier(); }
 void subgroupMemoryBarrierShared() { groupMemoryBarrier(); }
 void subgroupMemoryBarrierImage() { groupMemoryBarrier(); }
 #endif
+#ifndef GL_KHR_shader_subgroup_ballot
+bool subgroupInverseBallot(uvec4 value, uint index)
+{
+return any(notEqual(value & gl_SubgroupEqMask, uvec4(0u)));
+}
+uint subgroupBallotInclusiveBitCount(uvec4 value)
+{
+uvec4 v = value & gl_SubgroupLeMask;
+ivec4 c = bitCount(v);
+#ifdef GL_NV_shader_thread_group
+return uint(c.x);
+#else
+return uint(c.x + c.y + c.z + c.w);
+#endif
+}
+uint subgroupBallotExclusiveBitCount(uvec4 value)
+{
+uvec4 v = value & gl_SubgroupLtMask;
+ivec4 c = bitCount(v);
+#ifdef GL_NV_shader_thread_group
+return uint(c.x);
+#else
+return uint(c.x + c.y + c.z + c.w);
```

Comment:
actually you only need to do `c.x+c.y` because nobody specifies subgroups > 64 in GL extensions